src/store/Modules/Theme.ts

```diff
@@ -111,9 +111,10 @@ export default class Connections extends VuexModule {
       this.context.commit("addGlobalThemes", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("addGlobalThemes", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.